### PR TITLE
Fix broken links, typos, and other small errors

### PR DESCRIPTION
**Pull request by charislam**
# Description

Minor edits including:
- Fix broken links
- Fix typos
- Correct small errors
- Make small changes to align with style guide

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Fixes #[insert issue link, if any]

### CODE REVIEW

File: timescaledb/how-to-guides/data-tiering/move-data.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ another one).
 
 In addition, the [`move_chunk`][api-move-chunk] function has the
 ability to "reorder" the chunk during the migration in order to enable faster
-queries.  This behavior is similar to [`reorder_chunk`][api-reorder-chunk]; please
-see that documentation for more information.
+queries.  This behavior is similar to `reorder_chunk`. To learn more, see the
```
Comment (charislam) [inline timescaledb/how-to-guides/data-tiering/move-data.md:30]:
```suggestion
queries. This behavior is similar to `reorder_chunk`. To learn more, see the 
```

File: timescaledb/how-to-guides/compression/decompress-chunks.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Decompression
-If you need backfill or update data in a compressed chunk, you should decompress
+If you need to backfill or update data in a compressed chunk, you should decompress
```
Comment (Loquacity) [inline timescaledb/how-to-guides/compression/decompress-chunks.md:2]:
```suggestion
If you need to backfill or update data in a compressed chunk, decompress
```

File: timescaledb/how-to-guides/connecting/about-connecting.md [hunk 3]
```diff
@@ -19,30 +19,29 @@ To retrieve your connection details from a running Timescale Cloud service:
...
 <procedure>
 
 ### Finding connection details in Managed Service for TimescaleDB
 1.  Sign in to the [Managed Service for TimescaleDB portal][mst-portal].
 1.  In the `Services` tab, find the service you want to connect to, and check
     it is marked as `Running`.
-1.  Click the name of the service you want to connect to, to see the connection
-    information. Take a note of the `host`, `port`, and `password`.
+1.  Click the name of the service to see its connection information. Copy the
+    `host`, `port`, and `password`. You will need these when connecting.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/connecting/about-connecting.md:44]:
```suggestion
    `host`, `port`, and `password`. You need these to connect.
```

File: timescaledb/how-to-guides/connecting/about-psql.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # About psql
 The `psql` command line tool is widely used for interacting with a PostgreSQL or
 TimescaleDB instance, and it is available for all operating systems. Most of
-the instructions we give you assume you are using `psql`.
+the instructions in the Timescale docs assume you are using `psql`.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/connecting/about-psql.md:4]:
```suggestion
the instructions in the Timescale documentation assumes you are using `psql`.
```

File: timescaledb/how-to-guides/connecting/psql.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 # Install the psql connection tool
 The `psql` command line tool is widely used for interacting with a PostgreSQL or
 TimescaleDB instance, and it is available for all operating systems. Most of
-the instructions we give you assume you are using `psql`.
+the instructions in the Timescale docs assume you are using `psql`.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/connecting/psql.md:4]:
```suggestion
the instructions in the Timescale documentation assumes you are using `psql`.
```

**Review by Loquacity — APPROVED** [review]
Non-blocking suggestions

**Review by Rajakavitha1 — APPROVED** [review]

**Comment by Rajakavitha1** [comment]
@charislam I had no suggestions in addition to what  @Loquacity  has already made .

File: timescaledb/how-to-guides/connecting/about-psql.md (hunk 4 above)
Comment (charislam) [reply timescaledb/how-to-guides/connecting/about-psql.md:4]:
```suggestion
the instructions in the Timescale documentation assume you are using `psql`.
```

**Pull request merged**